models/task.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, Sequelize) => {
+  class Task extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+      this.hasMany(models.TaskComment, {
+        foreignKey: 'taskId',
+        targetKey: 'id'
+      });
+
+      this.belongsTo(models.Sprint, {
+        foreignKey: 'sprintId',
+        targetKey: 'id'
+      });
+
+      this.belongsTo(models.User, {
+        foreignKey: 'userId',
+        targetKey: 'id'
+      });
+    }
+  }
+  Task.init({
```

Comment:
add more columns like description, owners(like who is watching the task), deadline date etc.

Reply:
ok